.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
```

Comment:
can you have a look at other github ignores, too. you can use github template for the githubignore

Reply:
I have changed the . gitignore file using github template for Node.
https://github.com/github/gitignore/blob/main/Node.gitignore